build.sbt

```diff
@@ -47,7 +47,7 @@ lazy val microservice = Project(appName, file("."))
   .settings(defaultSettings(): _*)
   .configs(ItTest)
   .settings(inConfig(ItTest)(
-    Defaults.itSettings ++ headerSettings(ItTest) ++ automateHeaderSettings(ItTest) ++ ScalafmtPlugin.scalafmtConfigSettings): _*)
+    Defaults.testSettings ++ headerSettings(ItTest) ++ automateHeaderSettings(ItTest) ++ ScalafmtPlugin.scalafmtConfigSettings): _*)
```

Comment:
Optional but think it might be a good opportunity to match the integration test layout suggested by [sbt-settings](https://github.com/hmrc/sbt-settings?tab=readme-ov-file#integration-tests).   Refer to Scala 3 spike if you want to try it